docker/bash.sh

```diff
@@ -59,6 +59,17 @@ if [ "$#" -lt 1 ]; then
     exit -1
 fi
 
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+WORKSPACE="$(pwd)"
+
+if [ "$1" == "--repo-mount-point" ]; then
```

Comment:
It would be good to add this on line 23 small documentation to the command line, also to show the order in which parameters need to be provided.

Reply:
done